pkg/collector/corechecks/cluster/ksm/kubernetes_state_test.go

```diff
@@ -487,34 +487,6 @@ func TestProcessMetrics(t *testing.T) {
 				},
 			},
 		},
-		{
-			name:   "phase tag for ns",
```

Comment:
This test case is moved to "sumValuesAggregator aggregates namespace.count" in `pkg/collector/corechecks/cluster/ksm/kubernetes_state_aggregators_test.go`.